dbaas/dbaas.go

```diff
@@ -101,3 +110,73 @@ func GenRandString(ln int) string {
 
 	return string(b)
 }
+
+// GetPlatformType is for determine and return platform type
+func GetPlatformType() (PlatformType, error) {
+	var platform PlatformType
+
+	minikube, err := checkMinikube()
+	if err != nil {
+		return platform, err
+	}
+	if minikube {
+		return Minikube, nil
+	}
+
+	openshift, err := checkOpenshift()
```

Comment:
Just in case. Doesn't Minishift contain "openshift" in the version? If so, the Minishif check should go before the Openshift one